examples/src/main/scala/org/apache/spark/examples/streaming/KafkaWordCount.scala

```diff
@@ -65,6 +66,47 @@ object KafkaWordCount {
   }
 }
 
+/**
+  * Consumes messages from one or more topics in Kafka and does wordcount.
+  * Usage: NewKafkaWordCount <brokers> <group> <topics> <numThreads>
```

Comment:
This usage appears to be copied from the previous class but the name has changed